internal/services/servicesmocks/rpc_service_mocks.go

```diff
@@ -0,0 +1,27 @@
+package servicesmocks
+
+import (
+	"github.com/stellar/wallet-backend/internal/entities"
+	"github.com/stellar/wallet-backend/internal/services"
+	"github.com/stretchr/testify/mock"
+)
+
+type RPCServiceMock struct {
+	mock.Mock
+}
+
+var _ services.RPCService = (*RPCServiceMock)(nil)
+
+func (r *RPCServiceMock) SendTransaction(transactionXdr string) (entities.RPCSendTransactionResult, error) {
+	args := r.Called(transactionXdr)
+	return args.Get(0).(entities.RPCSendTransactionResult), args.Error(1)
+}
+
+func (r *RPCServiceMock) GetTransaction(transactionHash string) (entities.RPCGetTransactionResult, error) {
+	args := r.Called(transactionHash)
+	return args.Get(0).(entities.RPCGetTransactionResult), args.Error(1)
+}
+
+type TransactionManagerMock struct {
+	mock.Mock
+}
```

Comment:
Can this one be cleaned up? I see it's not being used

Reply:
Yeap removed